openedx/features/edly/tests/factories.py

```diff
@@ -38,10 +38,25 @@ class Meta(object):
     name = Sequence('Edly SubOrganization {}'.format)
     slug = Sequence('edly-sub-organization-{}'.format)
     edly_organization = SubFactory(EdlyOrganizationFactory)
-    edx_organization = SubFactory(OrganizationFactory)
     lms_site = SubFactory(SiteFactory)
     studio_site = SubFactory(SiteFactory)
 
+    @factory.post_generation
+    def edx_organization(self, create, extracted, **kwargs):
```

Comment:
since many to many relation can we change `edx_organization` -> `edx_organizations`